NuGet.Docs/ndocs/Consume-Packages/Package-Restore.md

```diff
@@ -154,11 +154,11 @@ To avoid all these issues, it's recommended to migrate any project using MSBuild
 The process is as follows: 
 
 1. Close Visual Studio to avoid file potential file locks and conflicts.
-2. If using TFS:
-	a. Remove `nuget.exe` and `nuget.targets` from the solution's `.nuget` folder and remove those files from the solution workspace.
-	b. Retain `nuget.config` with the `disableSourceControlIntegration` setting as explained in [Omitting packages with Team Foundation Version Control](/ndocs/consume-packages/packages-and-source-control#omitting-packages-with-team-foundation-version-control-(tfvc).
-3. If not using TFS:
-	a. Remove the `.nuget` folder from the solution and the solution workspace.
+2. If using TFS:  
+	a. Remove `nuget.exe` and `nuget.targets` from the solution's `.nuget` folder and remove those files from the solution workspace.  
+	b. Retain `nuget.config` with the `disableSourceControlIntegration` setting as explained in [Omitting packages with Team Foundation Version Control](/ndocs/consume-packages/packages-and-source-control#omitting-packages-with-team-foundation-version-control-(tfvc)).  
+3. If not using TFS:  
```

Comment:
Please use this link
/ndocs/consume-packages/packages-and-source-control#omitting-packages-with-team-foundation-version-control